packages/dds/tree/src/core/tree/chunk.ts

```diff
@@ -15,12 +15,31 @@ import {
 } from "./cursor.js";
 import { rootFieldKey } from "./types.js";
 
+/**
+ * Properties for incremental summarization of the chunks. This is used to determine if this chunk has changed since
+ * the last summary.
+ * If it has changed, the chunk will be summarized.
+ * If it hasn't changed, the state will be used to incrementally summarize it by referring to the summary for this
+ * chunk in the previous summary.
+ */
+export interface IncrementalSummaryProps {
+	/**
+	 * The reference ID of the summary for this chunk in the previous summary.
+	 */
+	readonly summaryRefId: string | undefined;
+	/**
+	 * Update the reference ID for this chunk in the current summary. This will be used in the next summary to refer to
+	 * this chunk if it has not changed since the last summary.
+	 */
+	updateSummaryRefId: (summaryRefId: string) => void;
```

Comment:
Is making this a reassign able bound function the intention? The way this is declared implies users can reassing it, and call it without providing the IncrementalSummaryProps as the this paramater.

Given neither of these seems intended based on the implementation I think this should be a method instead:

```suggestion
	updateSummaryRefId(summaryRefId: string): void;
```